### PR TITLE
Validation spec

### PR DESCRIPTION
**Pull request by lengushuru**
### Hello CR 👋 
In this PR The following changes to base are suggested :
_**Added the following validations**_

**For the User model:**

- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.

**For the Post model:**

- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

**Add unit specs for all of your models' methods and validations.**

### CODE REVIEW

File: spec/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'i am developer',
+                     posts_counter: 0)
+  subject = user.posts.new(title: 'Post 1', text: 'This is post 1', comments_counter: 0, likes_counters: 0,
+                           author_id: 11)
+
```
Comment (Deepakdanger) [inline spec/post_spec.rb:8]: - [ ] Kindly add some +ve test cases.

File: spec/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', posts_counter: 0) }
+
+  before { subject.save }
+
```
Comment (Deepakdanger) [inline spec/user_spec.rb:7]: - [ ] Kindly add some +ve test cases.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @lengushuru , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/user_spec.rb (hunk 2 above)
Comment (lengushuru) [reply spec/user_spec.rb:7]: Hello @Deepakdanger I am not understanding what you mean by +ve test cases

**Review by AnselemOdims — APPROVED** [review]
### Hi  @lengushuru 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**